test/e2e/jobscheduling.go

```diff
@@ -0,0 +1,225 @@
+/*
```

Comment:
(No action required) Most of this file is duplicative for the tests of replica scheduling, and I'd appreciate seeing the 2 converge in subsequent PRs.